domains/certificates/Query.API/API.AppTests/SwaggerTests.cs

```diff
@@ -9,11 +11,21 @@
 namespace API.AppTests;
 
 [UsesVerify]
-public class SwaggerTests : IClassFixture<QueryApiWebApplicationFactory>
+public class SwaggerTests : IClassFixture<QueryApiWebApplicationFactory>, IClassFixture<RabbitMqContainer>
```

Comment:
The code has now been changed in the factory to not wait for the bus, if RabbitMqSetup is null. So can we remove the RabbitMqContainer here from this test now?

Reply:
Seems some of the test still want to spin up rabbitmq. How is the lifecycle if these things?